### ISSUE TITLE
Search causes "Something went wrong" page

### ISSUE THREAD

**Issue by aneurinprice** [manyfold3d/manyfold#2439]
**Describe the bug**
Searching for anything causes a fail page (500)

**To Reproduce**
1. Enter text into search bar
2. Search
3. Observe

**Expected behavior**
Search should complete

**Screenshots**

![image](https://github.com/user-attachments/assets/bb02cec4-8eb1-4ffe-b915-be3e0584f424)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Firefox
 - Version: 28.0.2

**Additional context**
Application does not log this failiure. Only happened since the 0.74.0 upgrade.

**Comment by Floppy** [comment]
Oops - thank you, I'll fix that right away.

**Comment by Floppy** [comment]
OK, I thought I'd missed this completely but it doesn't happen in development. However, it does happen on my personal instance, so I can replicate it there.

The error log is:

```
I, [2024-07-25T08:35:10.876299 #247]  INFO -- : [1819aa39-244e-46a3-9130-0cd5d4f137b6] {"method":"GET","path":"/models","format":"html","controller":"ModelsController","action":"index","status":500,"allocations":6310,"duration":97.7,"view":0.0,"db":34.76,"@timestamp":"2024-07-25T08:35:10.875Z","@version":"1","message":"[500] GET /models (ModelsController#index)"}
F, [2024-07-25T08:35:10.877471 #247] FATAL -- : [1819aa39-244e-46a3-9130-0cd5d4f137b6]   
[1819aa39-244e-46a3-9130-0cd5d4f137b6] ActiveRecord::StatementInvalid (PG::InvalidColumnReference: ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list
LINE 1: ...test%') ORDER BY "creators"."name" ASC)) ORDER BY "models"."...
                                                             ^
):
[1819aa39-244e-46a3-9130-0cd5d4f137b6]   
```

**Comment by Floppy** [comment]
Ah, I think this for some reason doesn't happen on SQLite, just PostgreSQL. Hence not doing it in dev. Hm.

**Comment by Floppy** [comment]
Fixed in v0.74.1. Thanks for reporting it!
